autoload/nim/suggest.vim

```diff
@@ -41,8 +41,10 @@ function! s:NewInstance(project, file)
     endif
   endfunction
 
+  let s:nspath = exists('g:nvim_nim_nimsuggest_path') ? g:nvim_nim_nimsuggest_path : "nimsuggest"
```

Comment:
```suggestion
  let nscmd = exists('g:nim_nimsuggest_cmd') ? g:nim_nimsuggest_cmd : "nimsuggest"
```

There's no need to make this variable a script-local one. And I believe the name of the variable should reflects what it exactly is: The command used to invoke nimsuggest.

Also, currently the `nim_` prefix is used for configurables (`g:nim_fold`, `g:nim_nep1`), so please simplify it to just that.

Reply:
Done